src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/BuildAssetsUtils.cs

```diff
@@ -699,7 +700,7 @@ public static string GetLanguage(string nugetLanguage)
                         RootName = original.RootName,
                         Position = original.Position,
                         Items = original.Items,
-                        Conditions = original.Conditions.Concat(new[] { condition }).ToList()
+                        Conditions = original.Conditions.Concat(new[] { condition }).AsList()
```

Comment:
This change is unnecessary. There's no functional change here. 
LINQ generates enumerates which had to be converted to List.